etna/lib/etna/clients/metis/workflows/sync_metis_data_workflow.rb

```diff
@@ -82,20 +94,7 @@ def copy_file(dest:, url:, stub: false)
                 if start_time == end_time
                   next
                 end
-
```

Comment:
Just a thought, but this has been useful in debugging ETLs....might be nice to put verbose logging like this behind some sort of `--debug` or `--verbose` flag in the future, if we find a need for it.

Reply:
Also not necessary for this PR...

Reply:
It seemed hard to follow in any case, especially with concurrency turned up, because it doesn't print file names, just rates. We could just put it behind logger&.debug and let them turn up the log level.

What I wish for is a nice progress bar here if using STDOUT, but I'm not sure how to deal with concurrency in this case.

Reply:
Oh good point. Perhaps would be nice to include file names along with the rate on the log line. Yeah, progress bar with concurrency seems difficult ...